src/python/dbs/apis/dbsClient.py

```diff
@@ -19,6 +19,19 @@ def fixUrlPath(path):
     path = '/'.join([i for i in arr[1].split("/") if i])
     return '{}://{}'.format(arr[0], path)
 
+def serverCode(data):
+    """
+    Extract server code from provided DBS error data
+    """
+    if isinstance(data, dict) and 'exception' in data:
+        dbsErr = data.get('error', {})
+        return dbsErr.get('code', 0)
+    if isinstance(data, list) and len(data) == 1 and 'exception' in data[0]:
```

Comment:
From this implementation, you always call this function with a dictionary data type, so this 2nd `if` statement could likely be removed. There is no harm in keeping it around though.

Reply:
I intend to keep it this way as other clients (outside of WMCore may use it too).